src/views/Login.vue

```diff
@@ -48,11 +53,21 @@ export default defineComponent({
   data() {
     return {
       username: "",
-      password: ""
+      password: "",
+      instanceUrl: ""
     };
   },
+  computed: {
+    ...mapGetters({
+      instanceUrlSaved: 'user/getInstanceUrl'
```

Comment:
`currentInstanceUrlSaved`

Reply:
Used the correct name.
Thank You for reviewing my code.